addons/slideshow/functions/fnc_createSlideshow.sqf

```diff
@@ -20,18 +20,21 @@
 #include "script_component.hpp"
 
 private ["_currentSlideshow", "_slidesAction", "_varString"];
-params ["_objects", "_controllers", "_images", "_names", "_duration"];
+params [
+    ["_objects", [], []],
```

Comment:
If you want to only accept arrays as an element in `params` then you have to use:
`["_objects", [], [[]]],`
The third element is an array containing examples of all available data types. `[]` would be the default value and means everything is accepted.